### PR TITLE
Add more aufs tests and implement Status

### PR DESCRIPTION
**Pull request by crosbymichael**

### CODE REVIEW

**Comment by vieux** [comment]
LGTM, ping @creack

File: graphdriver/aufs/aufs_test.go [hunk 1]
```diff
@@ -465,6 +465,106 @@ func TestDiffSize(t *testing.T) {
...
+	}
+
+	// Add a file to the diff path with a fixed size
+	size := int64(1024)
+
+	f, err := os.Create(path.Join(diffPath, "test_file"))
+	if err != nil {
+		t.Fatal(err)
+	}
+	f.Truncate(size)
```
Comment (creack) [inline graphdriver/aufs/aufs_test.go]: Maybe it would be interesting to make sure the truncate worked?

File: graphdriver/aufs/aufs_test.go [hunk 2]
```diff
@@ -465,6 +465,106 @@ func TestDiffSize(t *testing.T) {
...
+	}
+	if diffSize != size {
+		t.Fatalf("Expected size to be %d got %d", size, diffSize)
+	}
+
+	if err := d.Create("2", "1"); err != nil {
+		t.Fatal(err)
+	}
+
+	diffSize, err = d.DiffSize("2")
```
Comment (creack) [inline graphdriver/aufs/aufs_test.go:515]: maybe this could be only 1 if/elseif block?

**Comment by creack** [comment]
LGTM

**Pull request merged**